scripts/ci/build_fusion_and_test.sh

```diff
@@ -21,7 +21,7 @@
 # BUILD_TYPE - The type of build `release`, `optimize`, or `internal`
 
 case "$CPP_STD" in
-    98|11)
+    11)
```

Comment:
I wonder if this should eventually include 14/17 too...

Reply:
Yes that would be a good idea.  It would make moving to those feature standards easier.  Would also want to add stages to make sure new code stays comparable with those standards.

Reply:
May also want to add stages for debug builds too.  Right now we only build release.

Reply:
@tst-nfarah I updated the release notes

Reply:
you can ignore the two 7160003.html files.  They will go away as soon as master is updated with two other PRs.  I'll make sure that those PRs are merged first.